src/backend/src/services/change-requests.services.ts

```diff
@@ -174,21 +174,85 @@ export default class ChangeRequestsService {
       // we don't want to have merge conflictS on the wbs element thus we check if there are unreviewed or open CRs on the wbs element
       await validateNoUnreviewedOpenCRs(foundCR.wbsElementId);
 
-      // must accept and review a change request before using the workpackage and project services
-      await prisma.change_Request.update({
-        where: { crId: foundCR.crId },
-        data: {
-          accepted: true,
-          dateReviewed: new Date()
-        }
-      });
-
       const associatedProject = foundCR.wbsElement.project;
       const associatedWorkPackage = foundCR.wbsElement.workPackage;
       const { wbsProposedChanges } = foundCR.scopeChangeRequest;
       const { workPackageProposedChanges } = wbsProposedChanges;
       const { projectProposedChanges } = wbsProposedChanges;
 
+      // must accept and review a change request before using the workpackage and project services
+      await prisma.scope_CR.update({
+        where: { changeRequestId: foundCR.crId },
+        data: {
+          changeRequest: {
+            update: {
+              accepted: true,
+              dateReviewed: new Date()
+            }
+          },
+          wbsOriginalData: {
+            create: {
+              name: foundCR.wbsElement.name,
+              status: foundCR.wbsElement.status,
+              leadId: foundCR.wbsElement.leadId,
+              managerId: foundCR.wbsElement.managerId,
+              links: {
+                connect: foundCR.wbsElement.links.map((link) => ({
+                  linkId: link.linkId
+                }))
+              },
+              projectProposedChanges: projectProposedChanges
+                ? {
+                    create: {
+                      budget: associatedProject!.budget,
+                      summary: associatedProject!.summary,
+                      rules: associatedProject!.rules,
+                      goals: {
+                        connect: associatedProject!.goals.map((goal) => ({ descriptionId: goal.descriptionId }))
```

Comment:
1. I wouldn't do the not null (!) if u can, it's generally not great bc it can let errors slip through, 2. I think this line needs to be updated to be projectQueryArgs, because goals for the original project is showing up as undefined when I try to review a Create Project CR and it's causing an error